<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuswahlSeite</class>
 <widget class="QWidget" name="AuswahlSeite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Futterkarre 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fdffe0;</string>
  </property>
  <widget class="QPushButton" name="btn_to_side_fu_heulage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>300</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #b0cfa5;
    color: #ffffff;
    border-radius: 18px;
    min-width: 300px;
    min-height: 120px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}
        </string>
   </property>
   <property name="text">
    <string>HEULAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_side_fu_heu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>300</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #b0cfa5;
    color: #ffffff;
    border-radius: 18px;
    min-width: 300px;
    min-height: 120px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}
        </string>
   </property>
   <property name="text">
    <string>HEU</string>
   </property>
  </widget>
  <widget class="QLabel" name="b_fuetterung">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F;
color: #fdffe0;
</string>
   </property>
   <property name="text">
    <string>Auswahlmenue</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_side_settings">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2E672F</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/Gear.png</normaloff>
     <normalon>:/icons/Gear.png</normalon>:/icons/Gear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_side_fu_laden">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>450</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #b0cfa5;
    color: #ffffff;
    border-radius: 18px;
    min-width: 300px;
    min-height: 120px;
}
QPushButton:hover {
    background-color: #8FBC8F;
    color: #f5f5b5;
}
        </string>
   </property>
   <property name="text">
    <string>KARRE LADEN</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
